deny.toml

```diff
@@ -100,4 +100,7 @@ skip = [
 
     # opentelemetry-otlp depends on an old version of tonic which depends on an old version of tokio-util
     { name = "tokio-util", version = "=0.6.10" },
+
+    # redis we’re using uses ancient sha
+    { name = "sha1", version = "=0.6.1" },
```

Comment:
is this related to this PR?

Reply:
Yes, though I’m not completely sure why to be honest.  `cargo build` decided that actix-http needs to be updated to 3.2 which changed its dependency from `sha-1` to `sha1 0.10.4` so now we depend on two different versions of `sha1`.  What I don’t understand is why Cargo decided to update actix-http.

Reply:
I’ve tracked it a little bit more.  What happens is conflict with zstd-sys.  actix-http and rocksdb both depend on it.  Updating RocksDB changes version requirement for that crate to ones which no longer work for actix-http 3.0 so it too needs to be updated.